### PR TITLE
How

### PR DESCRIPTION
**Pull request by eak000**

### CODE REVIEW

File: app/views/static_pages/how.html.erb [hunk 1]
```diff
@@ -19,9 +19,16 @@
...
+      Want to learn more or be a part of the she’s coding team? <%= link_to "Connect with us", "/connect" %>, or check out our project management systems: 
     </p>
   </div>
+  <div class="connect">
+     <ul class="connect_icons">
+      <li><%= link_to image_tag("static_pages/github_logo.png"), "https://github.com/shescoding",  title: 'Github', rel: 'tooltip', class: 'tooltips', target: "_blank" %><!-- <span>She's Coding on Github</span> --></li>
+      <li><%= link_to image_tag("static_pages/google_groups_logo.png"), "https://github.com/shescoding", title: 'Google Groups', rel: 'tooltip', target: "_blank" %></li>
+      <li><%= link_to image_tag("static_pages/trello_logo.png"), "https://github.com/shescoding", title: 'Trello', rel: 'tooltip', target: "_blank" %></li>
+      <li><%= link_to image_tag("static_pages/slack_logo.png"), "https://github.com/shescoding", title: 'Slack', rel: 'tooltip', target: "_blank" %></li>
```
Comment (natsteinmetz) [inline app/views/static_pages/how.html.erb]: Currently all the images link to Github, whereas they should link to their specific site.

**Comment by natsteinmetz** [comment]
@eak000 Could you try making the images a bit smaller? I think it looks really nice with the logos instead of the text as previously, but the logos could be approximately half the size of what they are now?

**Comment by eak000** [comment]
Yes to making them smaller, and I have NO CLUE how they are linking to github. I could swear I had them linking to the proper sites. I will fix that right now.

**Comment by natsteinmetz** [comment]
Also, @abillmaier I see that the images have grey borders at the bottom, could you replace them on trello with images that don't have this border, and notify Emily when they're up?
@eak000, can you exchange the pictures with whatever new ones Anna will post up, I would love to get rid of these grey bottom borders...

**Comment by natsteinmetz** [comment]
@eak000 Yeah, these things happen! :D

**Comment by eak000** [comment]
I made the changes, other than swapping out the newer images when they are ready.

**Comment by natsteinmetz** [comment]
Thanks Emily, and sorry for looking at these pull requests so late! Anna has never provided new images unfortunately, but yeah, it doesn't look too bad, and better than the status quo. So I'll merge this in as is.

**Pull request merged**